app/controllers/OneOffContributions.scala

```diff
@@ -38,21 +40,31 @@ class OneOffContributions(
     )
   }
 
-  def displayForm(paypal: Option[Boolean]): Action[AnyContent] = CachedAction() { implicit request =>
-    Ok(
-      oneOffContributions(
-        title = "Support the Guardian | One-off Contribution",
-        id = "oneoff-contributions-page",
-        js = "oneoffContributionsPage.js",
-        payPalButton = paypal.getOrElse(true),
-        defaultStripeConfig = stripeConfigProvider.get(false),
-        uatStripeConfig = stripeConfigProvider.get(true),
-        contributionsStripeEndpoint = contributionsStripeEndpoint,
-        contributionsPayPalEndpoint = contributionsPayPalEndpoint
-      )
+  def getOneOffContributions(idUser: Option[IdUser], paypal: Option[Boolean])(implicit request: RequestHeader): Html = {
```

Comment:
Generally I don't think that `get` adds much to a method name, and since the class is called `OneOffContributions` that isn't adding much either.  Possibly `formHtml` might be better?

Reply:
👍